docs/usage/java.md

```diff
@@ -103,6 +103,8 @@ Any repository URLs found within will be added as `registryUrls` to extracted de
 The manager for Gradle makes use of the `maven` datasource.
 Renovate can be configured to access more repositories and access repositories authenticated.
 
+### Artifactory
```

Comment:
The following example uses artifactory for demonstration purposes but the renovate config itself is not artifactory-specific. Also the second one to specify custom `registryUrls` is unrelated to artifactory. 

In other words, please drop this heading here as it may lead to confusion.